### DISCUSSION TITLE
UMPV - Way to turn off remembering pause state when new media loaded

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#14701] [Q&A]
Linux Debian based distro / GTK based Desktop Environment
MPV 0.38

When using UMPV I noticed that if you have it set to "replace", if you pause the media and then open up a different media item to replace the current media item, that new media item is in the paused state as well.  
I was wondering if anyone knew of a way to turn this off?
Meaning the media that replaces the existing paused media starts off playing, not paused.

Thank You

**Comment by advice2020** [comment]
Is this just the way that UMPV has to work due to what it is accomplishing?
The pause and play states have to be carried over into the next media item that is replacing it.

**Comment by advice2020** [comment]
Does anyone know if there are any settings that forces MPV to automatically "play" when next item is loaded?

**Comment by mrfragger** [comment]
https://github.com/jgreco/mpv-scripts/blob/master/unpauser.lua

**Comment by guidocella** [comment]
`--reset-on-next-file=pause`
